src/structure/pure/ImageTrigger.js

```diff
@@ -15,100 +15,181 @@
+ *     <caption>Image changed by incoming data 'blue'</caption>
+ *     <data address="0/0/0">blue</data>
+ *   </screenshot>
+ *   <screenshot name="image_trigger_changes">
+ *     <caption>Image changed by incoming data 'grey'</caption>
+ *     <data address="0/0/0">grey</data>
+ *   </screenshot>
+ * </meta>
+ * <imagetrigger src="icon/CometVisu_" suffix="png" sendValue="clicked" type="select" width="45px" height="32px">
+ *   <layout colspan="1" />
+ *   <address transform="DPT:16.001" mode="readwrite">0/0/0</address>
+ * </imagetrigger>
+ *
+ * @widget_example <meta>
+ *   <caption>Disable layout width by settings it to '0', to have widget with == image width</caption>
+ *   <screenshot name="image_trigger_colspan0">
+ *     <data address="0/0/0">1</data>
+ *   </screenshot>
+ * </meta>
+ * <imagetrigger src="icon/CometVisu_orange" suffix="png" sendValue="clicked" type="show" width="45px" height="32px">
+ *   <layout colspan="0" />
+ *   <address transform="DPT:1.001" mode="readwrite">0/0/0</address>
+ * </imagetrigger>
+ *
+ * @module structure/pure/ImageTrigger
+ * @requires structure/pure
  * @author Christian Mayer
- * @since 2012
+ * @since 0.8.0 (2012)
  */
 define( ['_common'], function( design ) {
   "use strict";
   var basicdesign = design.basicdesign;
   
   design.basicdesign.addCreator('imagetrigger', {
-  create: function( element, path, flavour, type ) { 
-    var 
-      $e = $(element),
-      classes = basicdesign.setWidgetLayout( $e, path );
-    classes += ' imagetrigger';
-    if( $e.attr('flavour') ) flavour = $e.attr('flavour');// sub design choice
-    if( flavour ) classes += ' flavour_' + flavour;
-    var ret_val = '<div class="widget clearfix image '+classes+'">';
-    ret_val += basicdesign.extractLabel( $e.find('label')[0], flavour );
-    var address = basicdesign.makeAddressList($e);
-    var layout = basicdesign.parseLayout( $e.children('layout')[0], {width:'100%'} );
-    var style = $.isEmptyObject(layout) ? '' : 'style="' + basicdesign.extractLayout( layout, type ) + '"';
+    /**
+     * Creates the widget HTML code
+     *
+     * @method create
+     * @param {Element} element - DOM-Element
+     * @param {String} path - internal path of the widget
+     * @param {String} flavour - Flavour of the widget
+     * @param {String} type - Page type (2d, 3d, ...)
+     * @return {String} HTML code
+     */
+    create: function( element, path, flavour, type ) {
+      var
+        $e = $(element),
+        classes = basicdesign.setWidgetLayout( $e, path );
+      classes += ' imagetrigger';
+      if( $e.attr('flavour') ) flavour = $e.attr('flavour');// sub design choice
+      if( flavour ) classes += ' flavour_' + flavour;
+      var ret_val = '<div class="widget clearfix image '+classes+'">';
+      ret_val += basicdesign.extractLabel( $e.find('label')[0], flavour );
+      var address = basicdesign.makeAddressList($e, null, path);
+      var width = $e.attr('width') || "100%";
```

Comment:
It's really hard to do a review when multiple things are changed at the same time, like a feature enhancement in this documentation PR.

=> I'm just assuming now that this will work and are not explicitly checking it.